conda_gitenv/deploy.py

```diff
@@ -136,6 +152,9 @@ def deploy_repo(repo, target):
 def configure_parser(parser):
     parser.add_argument('repo_uri', help='Repo to deploy.')
     parser.add_argument('target', help='Location to deploy the environments to.')
+    parser.add_argument('env_labels', nargs='*',  default=None, 
+                        help='Specific labelled environments to deploy, in the '
+                             'form "{environment}/{label}". Optional argument.',)
```

Comment:
Annoyingly this doesn't come up in the help as an optional argument, 
i.e.
```
$ python deploy.py --help
usage: deploy.py [-h] repo_uri target [env_labels [env_labels ...]]

Deploy the tracked environments.

positional arguments:
  repo_uri    Repo to deploy.
  target      Location to deploy the environments to.
  env_labels  Specific labelled environments to deploy, in the form
              "{environment}/{label}". Optional argument.

optional arguments:
  -h, --help  show this help message and exit
```
 unless you do
```parser.add_argument('--env_labels', nargs='*',  default=None ...```
which I don't want to do, as that means you have to do
```conda gitenv deploy /repo/to/deploy /location/to/deploy/to --env_labels default/current preview/current```
where I would prefer
```conda gitenv deploy /repo/to/deploy /location/to/deploy/to default/current preview/current```